src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -372,99 +388,34 @@ internal async Task<(bool, SumType<CompletionItem[], CompletionList>?)> TryGetPr
         }
 
         // In cases like "@{" preselection can lead to unexpected behavior, so let's exclude it.
-        private SumType<CompletionItem[], CompletionList> DoNotPreselect(SumType<CompletionItem[], CompletionList> completionResult)
+        private void DoNotPreselect(CompletionList completionList)
```

Comment:
Maybe `RemovePreselection`? I dono, I might just be in the mood the rename things.